modules/admin_manual/pages/release_notes.adoc

```diff
@@ -30,6 +31,130 @@
+- minimal bundle for the Server and required components, semantically versioned (`ownCloud-10.5.0`)
+- complete bundle for the Server and all supported apps, including the Enterprise features, not semantically versioned as it always contains the latest versions of all supported apps (`ownCloud-complete-<date>`)
+
+The availability of Enterprise features now only depends on the license key.
+As part of this process, the former `enterprise_key` app has been deprecated and is not shipped anymore. New versions for all Enterprise apps have been released and included in the Server 10.5 complete bundle. These are needed to work with PHP 7.4 and the new licensing mechanism (see below).
+
+TIP: **When upgrading to Server 10.5 it is mandatory to upgrade all apps to the latest versions as well to ensure compatibility with the supported PHP versions and the new licensing mechanism. Using the Server 10.5 complete bundle for upgrading you will get the latest versions of all apps.**
+
+TIP: It is mandatory to disable the deprecated `enterprise_key` app when upgrading. A migration step will do this automatically during the upgrade procedure.
+
+TIP: Practically, for existing installations the change does not make a big difference. Community installations will get all supported Community and Enterprise apps in addition but they will be disabled. For Enterprise installations the bundles stay equal. After upgrading it is recommended to check whether the desired apps are enabled/disabled.
+
+=== New Enterprise Trial Mechanism
+
+To facilitate the evaluation of ownCloud Enterprise functionality, Server 10.5 comes with a new trial mechanism. Previously, the process to upgrade from a Community installation to an Enterprise trial was not that easy.
+
+With Server 10.5 this process has been changed and made easier: Along with the changes to the Marketplace and in delivery, Enterprise functionality is available in every installation after upgrading or installing, respectively. The apps can be enabled like other apps which will start a 24-hour grace period if no valid license key is present. During this time the functionality can be fully used, even without a valid license key. To extend the period to 30 days, a demo license key can be obtained from a https://owncloud.com/try-enterprise/[new landing page]. The administrator will be informed and guided to obtain the demo license and can enter it in the web interface. If desired, the location of the landing page can be customized using the _config.php_ option `'grace_period.demo_key.link' => 'https://owncloud.com/try-enterprise/'`.
+
+=== New Admin UI to Supply License Keys
+
+As mentioned above, Server 10.5 adds new UI elements to set license keys in the _Admin -> General_ settings. Keys added this way will be stored in the database, not in _config.php_ as before. Still, there is legacy support taking into account when keys have been stored in _config.php_ or _license.config.php_.
```

Comment:
`license keys in the _Admin -> General_ settings` -->  `license keys in the menu:Settings[Admin > General] settings`